### ISSUE TITLE
 Passing custom variable(s) to Steam

### ISSUE THREAD

**Issue by THE-SIMPLE-MARK** [LeeviHalme/node-steam-openid#8]
There needs to be a way to pass a custom variable of the user's choice to the Steam API which it would pass back because **there is currently no way to verify if the id came from a specific user for example**.

**Comment by LeeviHalme** [comment]
Yes, that would be helpful. Added the enchantment tag to this issue and will fix it in my free time. While waiting, PRs are welcome :)

**Comment by joshuajeschek** [comment]
Hi, I've been just passing that in the redirect url
ik, it could be faked that way by the user, but I've just been using expiring uuids, so i think it's fine.
